File: week03-1.cpp
//week03-1.cpp C++
//LeetCode 2579.
class Solution {
public:
    long long coloredCells(int n) {
        long long ans=n; //要多寫這行，把數字升級，變成64位元，「很長很長的整數」
        return ans*ans+(ans-1)*(ans-1); //C/C++/Java/C#都要加分號
    }
};

File: week03/week03-1.py
# week03-1.py
# leetcode 2579.
class Solution:
    def coloredCells(self, n: int) -> int:
        return n**2+(n-1)**2
        #觀察題目的圖片(不看英文、不看中文)，頭轉45度，看到正方形